docs/docs/development/setup/native.mdx

```diff
@@ -0,0 +1,338 @@
+
+```sh
+pip install west
+```
+
+5. Initialize the application and update to fetch modules, including Zephyr:
+
+```sh
+west init -l app/
+west update
+```
+
+:::tip
+This step pulls down quite a bit of tooling, be patient!
+:::
+
+6. Export a [Zephyr CMake package](https://docs.zephyrproject.org/3.5.0/build/zephyr_cmake_package.html#cmake-pkg). This allows CMake to automatically load boilerplate code required for building Zephyr applications.
+
+```sh
+west zephyr-export
+```
+
+7. Install the additional dependencies found in Zephyr's `requirements-base.txt`:
+
+```sh
+pip install -r zephyr/scripts/requirements-base.txt
+```
+
+</TabItem>
+<TabItem value="glob">
+<OsTabs>
+<TabItem value="ubuntu">
+1. Install `west`, and make sure `~/.local/bin` is on your `PATH` environment variable:
+
+```sh
+pip3 install --user -U west
+echo 'export PATH=~/.local/bin:"$PATH"' >> ~/.bashrc
+source ~/.bashrc
+```
+
+2. Initialize the application and update to fetch modules, including Zephyr:
+
+```sh
+west init -l app/
+west update
+```
+
+:::tip
+This step pulls down quite a bit of tooling, be patient!
+:::
+
+3. Export a [Zephyr CMake package](https://docs.zephyrproject.org/3.5.0/build/zephyr_cmake_package.html#cmake-pkg). This allows CMake to automatically load boilerplate code required for building Zephyr applications.
+
+```sh
+west zephyr-export
+```
+
+4. Install the additional dependencies found in Zephyr's `requirements-base.txt`:
+
+```sh
+pip3 install --user -r zephyr/scripts/requirements-base.txt
+```
+
+</TabItem>
+<TabItem value="win">
+1. Install `west`, and make sure `~/.local/bin` is on your `PATH` environment variable:
+
+```sh
+pip install -U west
+```
+
+2. Initialize the application and update to fetch modules, including Zephyr:
+
+```sh
+west init -l app/
+west update
+```
+
+:::tip
+This step pulls down quite a bit of tooling, be patient!
+:::
+
+3. Export a [Zephyr CMake package](https://docs.zephyrproject.org/3.5.0/build/zephyr_cmake_package.html#cmake-pkg). This allows CMake to automatically load boilerplate code required for building Zephyr applications.
+
+```sh
+west zephyr-export
+```
+
+4. Install the additional dependencies found in Zephyr's `requirements-base.txt`:
+
+```sh
+pip install -r zephyr/scripts/requirements-base.txt
+```
+
+</TabItem>
+<TabItem value="mac">
+
+1. Install `west`
+
+```sh
+pip3 install -U west
+```
+
+2. Initialize the application and update to fetch modules, including Zephyr:
+
+```sh
+west init -l app/
+west update
+```
+
+:::tip
+This step pulls down quite a bit of tooling, be patient!
+:::
+
+3. Export a [Zephyr CMake package](https://docs.zephyrproject.org/3.5.0/build/zephyr_cmake_package.html#cmake-pkg). This allows CMake to automatically load boilerplate code required for building Zephyr applications.
+
+```sh
+west zephyr-export
+```
+
+4. Install the additional dependencies found in Zephyr's `requirements-base.txt`.
+
+```sh
+pip3 install -r ~/zephyrproject/zephyr/scripts/requirements-base.txt
```

Comment:
Is this the right path for macOS? I thought we'd want the one in `zephyr/scripts/requirements-base.txt` that comes with `west update`.

Reply:
Woops, nice catch.